### ISSUE TITLE
Blank screen after log in

### ISSUE THREAD

**Issue by meichthys** [zadam/trilium#1006]
TLDR: If you get a blank window after logging in, check to make javascript is enabled in your browser. This was observed in Edge/IE but may be relevant for other browsers as well.

Issue: After logging into the web interface using the latest versions of Edge or Internet Explorer, the page is blank. 

Steps to reproduce:

- Open IE/Edge
- Navigate to https://trilium.mydomain.org (it is behind an NGINX proxy with Let's encrypt certificate. Confirmed to be working in Firefox)
- Login with valid credentials

The page is completely blank:
![image](https://user-images.githubusercontent.com/10717998/81560760-08ae3700-9360-11ea-97e8-206688e5ff9a.png)

After looking at the page source, it is evident that Javascript is not enabled:
![image](https://user-images.githubusercontent.com/10717998/81561009-7f4b3480-9360-11ea-9203-20436f0e4770.png)

I'm not sure if javascript could be disabled by default on these browsers or if it was a local group policy on my pc. Either way i thought i'd report it just in-case someone else was confused by this.

Summary: make sure javascript is enabled :)

**Comment by meichthys** [comment]
Closing this issue since it's not an issue with Trilium.
@zadam Do you think it would be easy enough to display the following message on the page instead of in the page source only? 

"Trilium requires JavaScript to be enabled."

Very low priority obviously but may reduce some confusion in certain restricted or group-policy environments were javascript is disabled.

**Comment by zadam** [comment]
Hello, this has been fixed in the stable branch, thanks for reporting.
